src/index.js

```diff
@@ -30,18 +31,29 @@ const { index: bootstrapIndex } = createAlgoliaIndex(c.bootstrapIndexName);
 const stateManager = createStateManager(mainIndex);
 
 async function main() {
+  let start = Date.now();
   // first we make sure the bootstrap index has the correct settings
+  log.info('💪  Setting up Algolia');
   await setSettings(bootstrapIndex);
+  datadog.timing('main.init_algolia', Date.now() - start);
+
   // then we run the bootstrap
   // after a bootstrap is done, it's moved to main (with settings)
   // if it was already finished, we will set the settings on the main index
+  start = Date.now();
+  log.info('🗻  Bootstraping');
   await bootstrap(await stateManager.check());
+  datadog.timing('main.bootsrap', Date.now() - start);
```

Comment:
I was considering suggesting `process.hrtime(start)`, but that seems to have a lot more precision we don't need

Reply:
For the ref, in the end we can build this kind of dasboard 
![Screenshot 2019-08-03 14 39 47](https://user-images.githubusercontent.com/1637651/62412029-a1a76900-b5fc-11e9-9df6-5baa3645a989.png)